### PR TITLE
Feature/TT2-71 - Interpolate and move strings

### PR DESCRIPTION
**Pull request by kasgds**
This updates includes a template function to generate strings.

### CODE REVIEW

**Review by andylangridge — CHANGES_REQUESTED** [review]
Can we call the folder `constants` or (something else), instead of `i18n`? Couple of reasons for this:
1. I don't think we would every be localising log copy
2. In my opinion it's confusing to have things we _might_ need in future (YAGNI principal and all that)

**Comment by kasgds** [comment]
> Can we call the folder `constants` or (something else), instead of `i18n`? Couple of reasons for this:
> 
> 1. I don't think we would every be localising log copy
> 2. In my opinion it's confusing to have things we _might_ need in future (YAGNI principal and all that)

`constants` makes sense so I've renamed it to that.

File: src/utils/interpolateTemplate.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+interface MessagesInterface {
```
Comment (gdsandrewmccafferty) [inline src/utils/interpolateTemplate.ts:1]: I'd like to see some tests for the `interpolateTemplate` method, given that you've got a fair amount of logic in there, and everything else in the tests depend on this working properly.

Alternatively, you could put all the tests back to working how they did before (i.e. just assert on all the strings as before), and then you've got validation that we've not regressed.

**Review by andylangridge — DISMISSED** [review]

File: src/utils/interpolateTemplate.ts (hunk 1 above)
Comment (kasgds) [reply src/utils/interpolateTemplate.ts:1]: Fair point. I've undone the changes to the spec files.

**Review by gdsandrewmccafferty — APPROVED** [review]
Yep, better now that we've got the tests checking that the functionality works

**Pull request merged**